user-docs/documentation/docs/how-to-guides/5-deploy-cdk-constructs.md

```diff
@@ -0,0 +1,220 @@
+    super(scope, id);
+
+    const table = new Table(this, 'MyDynamodbTable', {
+      partitionKey: { name: 'PK', type: AttributeType.STRING },
+      sortKey: { name: 'SK', type: AttributeType.STRING },
+      billingMode: BillingMode.PAY_PER_REQUEST,
+    });
+
+    this.dynamodbArn = table.tableArn;
+    this.dynamodbName = table.tableName;
+  }
+}
+
+// Argument types are defined by MyConstruct's public string attributes
+const getCdkProperty = ServerlessCdkPlugin.getCdkPropertyHelper<MyConstruct>;
+
+const serverlessConfiguration: AWS & ServerlessCdkPluginConfig = {
+  service: "my-cdk-construct-service",
+
+  plugins: [
+    ...
+    '@swarmion/serverless-cdk-plugin',
+    ...
+  ],
+
+  provider: {...},
+  functions: {
+    myLambda: {
+      ...
+      environment: {
+        TABLE_NAME: getCdkProperty('dynamodbName'),
+        TABLE_ARN: getCdkProperty('dynamodbArn'),
+      },
+      ...
+    },
+  },
+  resources: {...},
+  construct: MyConstruct,
+};
+```
+
+## Export information from serverless to CDK
+
+### Using the serverless props
+
+The plugin exposes a construct wrapper to access the config file and the runtime serverless service object if you need it. The example below shows how to retrieve the stage value.
+
+```typescript
+import ServerlessCdkPlugin, {
+  ServerlessProps,
+  ServerlessCdkPluginConfig,
+} from '@swarmion/serverless-cdk-plugin';
+
+
+class MyConstruct extends ServerlessCdkPlugin.ServerlessConstruct {
+  constructor(scope: Construct, id: string, serverlessProps: ServerlessProps) {
+    super(scope, id);
+
+    new Table(this, 'MyDynamodbTable', {
+      tableName: `table-${serverlessProps.service.provider.stage}`,
+      partitionKey: { name: 'PK', type: AttributeType.STRING },
+      sortKey: { name: 'SK', type: AttributeType.STRING },
+      billingMode: BillingMode.PAY_PER_REQUEST,
+      removalPolicy:
+        serverlessProps.service.provider.stage === 'production'
+          ? RemovalPolicy.RETAIN
+          : RemovalPolicy.DESTROY,
+    });
+  }
+}
+
+const serverlessConfiguration: AWS & ServerlessCdkPluginConfig = {
+  service: "my-cdk-construct-service",
+
+  plugins: [
+    ...
+    '@swarmion/serverless-cdk-plugin',
+    ...
+  ],
+
+  provider: {...},
+  functions: {...},
+  resources: {...},
+  construct: MyConstruct, // You can provide either a Construct or a ServerlessConstruct here
+};
+```
+
+### Using serverless variables
+
+You can also use serverless variables directly in the CDK.
+
+```typescript
+class MyConstruct extends Construct {
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    new Table(this, 'MyDynamodbTable', {
+      tableName: 'table-${sls:stage}', // Use variables as if you were in the serverless config
+      partitionKey: { name: 'PK', type: AttributeType.STRING },
+      sortKey: { name: 'SK', type: AttributeType.STRING },
+      billingMode: BillingMode.PAY_PER_REQUEST,
+    });
+  }
+}
+```
+
+:::warning
```

Comment:
Lets put a `caution` instead of a `warning` here, the red is I think a bit too much